docs/application-development-guide.md

```diff
@@ -1830,29 +1830,26 @@ This section covers many of the features that are available to use in your [Temp
 
 ### Signals
 
-First define your Signal type.
-Then add a Signal handler to your Workflow Definition.
+First, Define the structure of your Signal.
```

Comment:
By analogy with our lowercasing of "signal" when used as a verb, should we lowercase "define" when used as a verb?